packages/.template/src/elements/{{capitalize component}}Text.tsx

```diff
@@ -8,11 +8,13 @@
 import React, { HTMLAttributes, forwardRef } from 'react';
 import { Styled{{capitalize component}}Text } from '../styled';
 
-/**
- * @extends HTMLAttributes<HTMLElement>
- */
-export const {{capitalize component}}Text = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>((props, ref) => (
+const {{capitalize component}}TextComponent = forwardRef<HTMLElement, HTMLAttributes<HTMLElement>>((props, ref) => (
```

Comment:
Is there a preference between `forwardRef` and `React.forwardRef` notation?

Reply:
I prefer `forwardRef` as it shortens the prettier line wrap and enhances readability/scanability. It is also the form documented in the MD.